src/syrupy/serializers/base.py

```diff
@@ -40,6 +43,19 @@ def dirname(self) -> str:
             return os.path.join(test_dirname, SNAPSHOT_DIRNAME, subdir_name)
         return os.path.join(test_dirname, SNAPSHOT_DIRNAME)
 
+    @property
+    def snapshot_subdirectory_name(self) -> Optional[str]:
```

Comment:
this is just grouping methods/fields that should be overridden together?

Reply:
Yeah essentially